pyvista/core/pointset.py

```diff
@@ -1062,11 +1068,36 @@ def n_strips(self) -> int:  # numpydoc ignore=RT01
         """
         return self.GetNumberOfStrips()
 
+    @staticmethod
+    def use_strict_n_faces(mode: bool) -> None:
+        """Global opt-in to strict n_faces.
+
+        Parameters
+        ----------
+        mode : bool
+            If true, all future calls to :attr:`n_faces <pyvista.PolyData.n_faces>`
+            will return the same thing as :attr:`n_faces_strict <pyvista.PolyData.n_faces_strict>`.
+
+        """
+        global _USE_STRICT_N_FACES
+        _USE_STRICT_N_FACES = mode
+
     @property
     def n_faces(self) -> int:  # numpydoc ignore=RT01
         """Return the number of cells.
 
-        Alias for ``n_cells``.
+        .. deprecated:: 0.43.0
+        The current (deprecated) behavior of this property is to
+        return the total number of cells, i.e. the sum of the number of
+        vertices, lines, triangle strips, and polygonal faces.
+        In the future, this will change to return only the number of
+        polygonal faces, i.e. those cells represented in the
+        `pv.PolyData.faces` array. If you want the total number of cells,
+        use `pv.PolyData.n_cells`. If you want only the number of polygonal faces,
+        use `pv.PolyData.n_faces_strict`. Alternatively, you can opt into the
+        future behavior globally by calling `pv.PolyData.use_strict_n_faces(True)`,
+        in which case `pv.PolyData.n_faces` will return the same thing as
+        `pv.PolyData.n_faces_strict`.
```

Comment:
```suggestion
        .. deprecated:: 0.43.0
            The current (deprecated) behavior of this property is to
            return the total number of cells, i.e. the sum of the number of
            vertices, lines, triangle strips, and polygonal faces.
            In the future, this will change to return only the number of
            polygonal faces, i.e. those cells represented in the
            `pv.PolyData.faces` array. If you want the total number of cells,
            use `pv.PolyData.n_cells`. If you want only the number of polygonal faces,
            use `pv.PolyData.n_faces_strict`. Alternatively, you can opt into the
            future behavior globally by calling `pv.PolyData.use_strict_n_faces(True)`,
            in which case `pv.PolyData.n_faces` will return the same thing as
            `pv.PolyData.n_faces_strict`.
```